api/MouseEvent.json

```diff
@@ -65,7 +65,7 @@
               "version_added": "47"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "12"
```

Comment:
I confirmed that `new MouseEvent('bla')` works in Edge 15.

I also confirmed http://mdn-bcd-collector.appspot.com/tests/api/MouseEvent fails in IE11 with "Object doesn't support this action" so no IE update needed. Good!